files/centos-8/ec2_dev_2_volid.py

```diff
@@ -0,0 +1,70 @@
+import requests
```

Comment:
Could you add a comment describing where this file come from?
Which are the differences with the default one?

Reply:
Done